appengine/flexible/django_cloudsql/mysite/settings.py

```diff
@@ -164,6 +164,7 @@
 
 # Static files (CSS, JavaScript, Images)
 # [START staticurl]
+# [START gaeflex_py_django_static_config]
 # [START gaeflex_py_django_staticconfig]
```

Comment:
does `staticconfig` tag still need to be here? Or do you plan on removing that in a future PR once `static_config` region tag fully replaces it?

Reply:
it does for now, until I replace it with the new version in the docs, then I can remove it from the sample in another PR

Reply:
Sounds good! Thank you for clarifying it.